sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/codegen/BufferHolder.java

```diff
@@ -31,24 +31,24 @@
  * for each incoming record, we should call `reset` of BufferHolder instance before write the record
  * and reuse the data buffer.
  *
- * Generally we should call `UnsafeRow.setTotalSize` and pass in `BufferHolder.totalSize` to update
+ * Generally we should call `UnsafeRowWriter.setTotalSize` using `BufferHolder.totalSize` to update
```

Comment:
Not sure if this description is still here or better to move to `UnsafeRowWriter`.

Reply:
+1, now BufferHolder is hidden behind `UnsafeWriter`